video_xblock/static/js/studio-edit/transcripts-manual-upload.js

```diff
@@ -240,11 +240,14 @@ function createTranscriptBlock(langCode, langLabel, transcriptsValue, downloadTr
  */
 function clickUploader(event, $fileUploader) {
     'use strict';
+    var handoutsAllowedFileTypes = '.png, .jpeg, .jpg, .gif, .tiff';
```

Comment:
Please check what types support stock open edx video xmodule.
I'm certain this list should include `.pdf`, and maybe some more.